tests/freebsd_port.h

```diff
@@ -0,0 +1,54 @@
+#include <limits.h>
```

Comment:
seems a lot is like the linux port header, same as other comments, should be more of a "non_darwin_port.h"